guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
<blockquote>attribute data from feature layers and <code><strong>DataFrame</strong></code><strong>s.</strong></blockquote><blockquote>remove space in the word DataFrames in the above line</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>